Already in debcargo-conf.git, can be deleted for next source import.

From 0f37ef7d8dfe7b65fa2e8b1d294fc8204bc973b4 Mon Sep 17 00:00:00 2001
From: Ximin Luo <infinity0@pwned.gg>
Date: Sat, 23 Oct 2021 23:51:59 +0100
Subject: [PATCH 2/2] Revert "Bump libgit2 submodule to 1.2.0 (#744)"

This reverts commit c273ed95357f39bea4234bc5880fd4bd6ff66e8b.
---
 libgit2-sys/lib.rs | 20 --------------------
 1 file changed, 20 deletions(-)

diff --git a/libgit2-sys/lib.rs b/libgit2-sys/lib.rs
index 044e593..eb99d1e 100644
--- a/vendor/libgit2-sys/lib.rs
+++ b/vendor/libgit2-sys/lib.rs
@@ -353,8 +353,6 @@ pub type git_indexer_progress_cb =
 )]
 pub type git_transfer_progress = git_indexer_progress;
 
-pub type git_remote_ready_cb = Option<extern "C" fn(*mut git_remote, c_int, *mut c_void) -> c_int>;
-
 #[repr(C)]
 pub struct git_remote_callbacks {
     pub version: c_uint,
@@ -370,7 +368,6 @@ pub struct git_remote_callbacks {
     pub push_update_reference: git_push_update_reference_cb,
     pub push_negotiation: git_push_negotiation,
     pub transport: git_transport_cb,
-    pub remote_ready: git_remote_ready_cb,
     pub payload: *mut c_void,
     pub resolve_url: git_url_resolve_cb,
 }
@@ -1490,8 +1487,6 @@ pub struct git_odb_backend {
         ) -> c_int,
     >,
 
-    pub writemidx: Option<extern "C" fn(*mut git_odb_backend) -> c_int>,
-
     pub freshen: Option<extern "C" fn(*mut git_odb_backend, *const git_oid) -> c_int>,
 
     pub free: Option<extern "C" fn(*mut git_odb_backend)>,
@@ -1751,20 +1746,6 @@ pub type git_commit_signing_cb = Option<
     ) -> c_int,
 >;
 
-pub type git_commit_create_cb = Option<
-    extern "C" fn(
-        *mut git_oid,
-        *const git_signature,
-        *const git_signature,
-        *const c_char,
-        *const c_char,
-        *const git_tree,
-        usize,
-        *const git_commit,
-        *mut c_void,
-    ) -> c_int,
->;
-
 pub const GIT_REBASE_NO_OPERATION: usize = usize::max_value();
 
 #[repr(C)]
@@ -1775,7 +1756,6 @@ pub struct git_rebase_options {
     pub rewrite_notes_ref: *const c_char,
     pub merge_options: git_merge_options,
     pub checkout_options: git_checkout_options,
-    pub commit_create_cb: git_commit_create_cb,
     pub signing_cb: git_commit_signing_cb,
     pub payload: *mut c_void,
 }
-- 
2.33.0
